com.gitlab.kendellfab.restscope.json

```diff
@@ -0,0 +1,55 @@
+{
+    "app-id": "com.gitlab.kendellfab.restscope",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.dmd"
+    ],
+    "command": "restscope",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=home"
```

Comment:
Does your application need access to all of home?

Reply:
Perhaps there is a better way than I am aware of.  But my use case, seemed to warrant having access to the home directory.  

Part of wanting to build this, is I wanted a less proprietary method to sync my requests (where postman et.al. have their own method of syncing requests), I wanted to have a flat file (sqlite in this case) that I could save all of my requests to and use nextcloud/drop box/other sync service to sync my requests with.  

Not knowing before hand which service a user could be using, having access to their home directory seemed like a good fit.

Reply:
You could use Gtk.FileChooserNative for that, it uses the documents portal

Reply:
Or at least a file under `$XDG_DATA_HOME` which is available by default and points to `~/.var/app/appid/data` directory.

Reply:
I've been working on adding the Gtk.FilechooserNative to open my database files, without access to my home directory within flatpak.  I'm noticing though, that I'm unable to write to my database file, when needed.  It appears to be an issue with opening the file through the portal, as the running outside of flatpak works fine.

Do any of you have any input on this issue?  As I'm still very new to flatpak.

Reply:
From what I can gather on the subject, sqlite throws that error when the associated journal file isn't able to be written to.  Since that file is created alongside the main database file and the database file is the only one that was given permission to when opening, how would I be able to get access to the journal file as well?

Reply:
It's a known problem, see https://github.com/flatpak/xdg-desktop-portal/issues/463

Reply:
So I tried the app and it seems to hardcode the config path to `$HOME/.config` which isn't right. You should use glib's `g_get_user_data_dir` instead which should be `$HOME/.var/app/$app-id/data`  under Flatpak.